app/src/main/java/com/example/campusguide/directions/ChooseDestinationOptions.kt

```diff
@@ -0,0 +1,84 @@
+package com.example.campusguide.directions
+
+import android.graphics.Color
+import android.graphics.drawable.ColorDrawable
+import android.location.Location
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import androidx.fragment.app.DialogFragment
+import com.example.campusguide.Constants
+import com.example.campusguide.MapsActivity
+import com.example.campusguide.R
+import com.example.campusguide.search.CustomSearch
+import com.example.campusguide.search.SearchLocation
+import com.example.campusguide.search.SearchLocationListener
+import com.example.campusguide.search.indoor.BuildingIndexSingleton
+import com.example.campusguide.search.indoor.IndoorLocationProvider
+import com.example.campusguide.search.outdoor.PlacesApiSearchLocationProvider
+import com.google.android.libraries.places.api.Places
+import com.google.android.libraries.places.api.model.Place
+import com.google.android.libraries.places.api.net.FetchPlaceRequest
+import com.google.android.libraries.places.api.net.PlacesClient
+
+class ChooseDestinationOptions(private val locationSelectedListener: (location: Location) -> Unit): DialogFragment() {
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        dialog?.window?.setBackgroundDrawable(ColorDrawable(Color.TRANSPARENT))
+        val view = inflater.inflate(R.layout.choose_destination_options, container, false)
+
+        view.findViewById<Button>(R.id.calendar).setOnClickListener {
+            useNextEvent()
+        }
+        view.findViewById<Button>(R.id.fromMap).setOnClickListener {
+            chooseFromMap()
+        }
+        view.findViewById<Button>(R.id.search).setOnClickListener {
+            searchForLocation()
+        }
+
+        return view
+    }
+
+    private fun useNextEvent(){
+        //TODO fill this method once we have the logic to use a calendar Event
+        dismiss()
+        val location = Location("Montreal")
+        location.latitude = 45.5017
```

Comment:
From my testing this dummy data will break the directions activity, i'd advise replacing it with something that is at least functional (unless i'm wrong about that)

Reply:
It should be functional now, I added a "-" in front of the latitude